build/scripts/CrossBranchPorting.psm1

```diff
@@ -35,6 +39,16 @@ function New-BCAppsBackport() {
         Write-Host "Pull Request Title: $($pullRequestDetails.title)" -ForegroundColor Cyan
         Write-Host "Pull Request Description: `n$($pullRequestDetails.body)" -ForegroundColor Cyan
 
+        $workItemNumber = "[**Insert Work Item Number Here**]" # By default, work item number is not set and has to be added manually in the PR description
+        if($ReuseWorkItem) {
+            if ($pullRequestDetails.body -match "AB#(\d+)") {
+                $workItemNumber = $matches[1]
+                Write-Host "Reusing work item number: $workItemNumber" -ForegroundColor Cyan
```

Comment:
Can we add something to this message discouraging people to use this option if their PR is a hotfix? With hotfixes we do want a different bug per branch as far as I know